package.json

```
@@ -68,7 +68,7 @@
   },
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.6",
-    "@asyncapi/specs": "^2.7.7",
+    "@asyncapi/specs": "git://github.com/asyncapi/asyncapi-node.git#security-scheme-for-kafka",
```

CI is complaining because `security-scheme-for-kafka` branch doesn't exist. And it's right, it's only in your fork 😄 I think you should probably point to the release branch instead:

```suggestion
    "@asyncapi/specs": "git://github.com/asyncapi/asyncapi-node.git#2021-06-release",
```

Sorry for the confusion.

---

I say just wait for the merge of PR on `asyncapi-node` and then use the released release candidate here